admin/Dockerfile

```diff
@@ -13,7 +13,7 @@ ENV BUILD_VERSION="0.0.0.0"
 ## We cannot do `$(git rev-parse --short HEAD)` here so we default to 0000000
 ENV BUILD_COMMIT="0000000"
 ## SET NODE_ENV
-ENV NODE_ENV="production"
+ARG NODE_ENV="production"
```

Comment:
@ulfgebhardt Can this cause any problems?

Reply:
Yes this can have side effects regarding `config/index.ts`